pysbagen/inspector.py

```
@@ -0,0 +1,278 @@
+            report.near_mono = bool(np.allclose(left, right, atol=1e-6))
+            report.stereo_correlation = 1.0 if report.near_mono else None
+
+
+def _inspect_with_ffprobe(report: AudioSourceReport, source: Path) -> None:
+    executable = shutil.which("ffprobe")
+    if executable is None:
+        report.warnings.append("ffprobe is not installed, so only file existence could be verified.")
+        return
+    command = [executable, "-v", "error", "-select_streams", "a:0", "-show_entries", "stream=codec_name,channels,sample_rate,duration,nb_frames:format=format_name,duration", "-of", "json", str(source)]
+    try:
+        payload = json.loads(subprocess.run(command, check=True, capture_output=True, text=True).stdout)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)